cmake/Modules/SynfigGettext.cmake

```diff
@@ -0,0 +1,70 @@
+#[=============================================================[
+
+Despite CMake providing functions to process .po files,
```

Comment:
Here is my addtion to comment:

Although CMake provides functions for processing .po files, it cannot be used because GETTEXT_PROCESS_PO_FILES installs files with an invalid name. For example: for an en.gmo file, we should get the ${LOCALE_DIR}/${language}/LC_MESSAGES/${project_name}.mo, not ${LOCALE_DIR}/${language}/LC_MESSAGES/en.mo, as it does, so we need to get the possibility to pass ${project_name} to that function and use it for file naming.
We also added an issue to CMake:
https://gitlab.kitware.com/cmake/cmake/-/issues/20792